config/initializers/assets.rb

```diff
@@ -9,8 +9,3 @@
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
 
-# Precompile additional assets (application.js.coffee.erb, application.css, and all non-JS/CSS are already added)
-Rails.application.config.assets.precompile += %w( diagram.js graphing.js map_marker.js ace.js )
```

Comment:
`sprocket-rails` now requires a manifest.js file which is used as a [configuration](https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb#L106) which assets should be precompiled.